src/TodoListItem.jsx

```diff
@@ -0,0 +1,7 @@
+const TodoListItem = (props) => {
```

Comment:
can be shorten to one line:
``const TodoListItem = (props) => <li key={props.id}>{props.todo}</li>;``